README.md

```diff
@@ -1,7 +1,7 @@
 # Library Developers
 
 ## Overview
-Use `browzine-summon-adapter` to enhance Summon search results with BrowZine data; Adds links to Journal and Article content in BrowZine; Uses BrowZine Journal covers. The BrowZine Summon adapter supports IE11+ and evergreen browsers (Chrome, Firefox, Safari, Microsoft Edge).
+Use `browzine-summon-adapter` to enhance Summon or 360 Core search results with BrowZine data; Adds links to Journal and Article content in BrowZine; Uses BrowZine Journal covers. The BrowZine Summon adapter supports IE11+ and evergreen browsers (Chrome, Firefox, Safari, Microsoft Edge).
```

Comment:
Two separate core classes: `browzine.summon` and `browzine.serSol360Core` so not so much worried about the core functionality. I do worry about them sharing the same MutationObserver instance however. But, I know we're thinking to have a separate class for each service, so when we do that, that should this particular risk because yes, one could affect the other at the MutationObserver integration point.

Reply:
Cool - hopefully it's not _too_ much extra work to accomplish!